### PR TITLE
Update operational layers doc

### PR DESCRIPTION
**Pull request by Oshratn**
Added search questions and a one-line summary that serves as an ending to the document.
Towards #13270

### CODE REVIEW

File: src/content/basics/giant-swarm-operational-layers/index.md [hunk 1]
```diff
@@ -82,6 +99,8 @@ However, a user with access to the Kubernetes API does not gain any permisssions
 
 This enables the customer to individually set up their user management according to the needs of their organization. The configuration for this can be kept in version control and needs to be done by an initial cluster admin user, which can be created by the Giant Swarm API user mentioned above.
 
+The Giant Swarm opertaional layers are the means which we use to keep a separation of concerns between different users of the Giant Swarm platform. This reducess burden from an operational perspective as well as enhancing security.
```
Comment (marians) [inline src/content/basics/giant-swarm-operational-layers/index.md:102]:
```suggestion
The Giant Swarm operational layers are the means which we use to keep a separation of concerns between different users of the Giant Swarm platform. This reduces burden from an operational perspective as well as enhancing security.
```

**Comment by marians** [comment]
Since this is a review, I want to question everything...

- Is it good to have our company name appear in the title and in the URL? I would prefer not to have that. (Likewise in prosa, I would rather not use the company name in a phrase like "The Giant Swarm operational layers...")
- Is "Operational Layers" actually a term that means anything specific to anybody? Can we improve?

File: src/content/basics/giant-swarm-operational-layers/index.md [hunk 2]
```diff
@@ -1,10 +1,27 @@
 ---
 title: Giant Swarm Operational Layers
 description: Here you learn how the operational layers of Giant Swarm are defined and what the intended operational model is.
-date: 2020-05-18
+date: 2020-11-18
 type: page
 weight: 80
 categories: ["basics"]
+last-review-date: 2020-11-18
```
Comment (marians) [inline src/content/basics/giant-swarm-operational-layers/index.md:8]:
```suggestion
last_review_date: 2020-11-18
```

**Comment by Oshratn** [comment]
> Is it good to have our company name appear in the title and in the URL? 

I have no strong feelings one way or another.

> Is "Operational Layers" actually a term that means anything specific to anybody? Can we improve?

A quick Google search shows that the term means many things to many people. That being said, I am not a software engineer, so this is just my impression.

A suggestion to solve your concerns about the usage of 'Giant Swarm Operational Layers' is to use the opening paragraph to define the term as we use it at Giant Swarm and then not need the `Giant Swarm` descriptor all over the place.

WDYT?

**Comment by marians** [comment]
> A suggestion to solve your concerns about the usage of 'Giant Swarm Operational Layers' is to use the opening paragraph to define the term as we use it at Giant Swarm and then not need the `Giant Swarm` descriptor all over the place.
> 
> WDYT?

Sounds wise. Thanks!

**Comment by Oshratn** [comment]
The term Giant Swarm Operational Layers is used once in the opening and once in the summary of the document.

**Comment by Oshratn** [comment]
@marians are we good to go on this?

**Comment by marians** [comment]
Yeah, sure. Not blocking.

**Comment by Oshratn** [comment]
So I need an approval 🙏

**Review by marians — APPROVED** [review]

**Pull request merged**